contracts/verifier/CcipResponseVerifier.sol

```diff
@@ -26,8 +26,35 @@ abstract contract CcipResponseVerifier is ICcipResponseVerifier, SupportsInterfa
      */
     address public owner;
 
+    /**
+     * @notice The graphql endpoint url
+     * The graphQlUrl is used to provide metadata of the resolver
+     */
     string public graphqlUrl;
 
+    /**
+     * @notice The resolver name
+     * The name of the resolver
+     */
+    string public resolverName;
+    /**
+     * @notice the chainId at which the resolver resolves data from
+     */
+    uint256 public l2ResolverChainID;
```

Comment:
Maybe add to the comments that `l2ResolverChainID` of 0 means that the data is retrieved from an other source like an off-chain signer